lti_consumer/models.py

```diff
@@ -65,6 +76,29 @@ class LtiConfiguration(models.Model):
         blank=True,
     )
 
+    # LTI 1.3 Related variables
+    _lti_1p3_platform_private_key = models.TextField(
```

Comment:
I see why there's a leading underscore, because there's a property for accessing it.  But this will make the db column have a leading underscore... ick.